pkg/prow/prow.go

```diff
@@ -162,3 +175,30 @@ func determineJobTarget(pjYAML *v1.ProwJob) (jobTarget string, err error) {
 	}
 	return "", fmt.Errorf("%s expected %+v to contain arg --target", errPrefix, args)
 }
+
+func determineJobTargetFromProwURL(prowURL string) (jobTarget string, err error) {
+	if strings.Contains(prowURL, "pull-ci-redhat-appstudio-infra-deployments") {
+		// prow URL is from infra-deployments repo
+		jobTarget = "appstudio-e2e-tests"
+	} else if strings.Contains(prowURL, "pull-ci-redhat-appstudio-e2e-tests") {
+		// prow URL is from e2e-tests repo
+		jobTarget = "redhat-appstudio-e2e"
```

Comment:
not sure if it's something consider as a part of this PR, but I'm thinking what would we do if we'd want to use this for other repos than infra-deployments and e2e-tests in a future (like build-service, integration-service etc).

one solution I can think of is to parse the html of prow job URL and get the prow job ID from the "a href", e.g.:
```html
<a href="/prowjob?prowjob=6e7855cd-776c-48fb-8029-161ca3c1916a" onclick="gtag('event', 'view_job_yaml', {event_category: 'engagement', transport_type: 'beacon'})">Prow Job YAML</a>
```
and then use `determineJobTargetFromYAML` func to get the job target, wdyt?

Reply:
I agree with you that the current method to find out the job target doesn't scale well. And I like the approach you suggested, it's hacky but would scale well.